lib/kitchen_sink/boolean.ex

```diff
@@ -0,0 +1,25 @@
+defmodule KitchenSink.Boolean do
+
+  @moduledoc """
+  this module is for Boolean functions
+  """
+
+  @doc """
+  Parse boolean either from string or integer, if it's not a boolean then it returns :error
+  """
+  def parse(st) when is_binary(st) do
+    String.downcase(st)
+    |> case do
+      "true" -> true
+      "false" -> false
```

Comment:
Let's expand these to https://github.com/prodis/wannabe_bool/blob/master/lib/wannabe_bool/symbol.rb